src/main/java/codeu/model/data/Conversation.java

```diff
@@ -26,6 +26,7 @@
   public final UUID owner;
   public final Instant creation;
   public final String title;
+  private Boolean isPrivate;
```

Comment:
Just curious why we want the wrapper class for boolean?

Reply:
Didn't think about it. Guess it's not necessary.